GetJobber/atlantis
# feat(components): localized date formatter

## Motivations

Now that we are starting to support localization, date and time formatters needs to be localized too

## Changes



### Added

- `formatTime` and `formatDate` in the `useAtlantisI18n` hook to automatically localized the date or time string depending on your settings in Atlantis Context

### Changed

- 

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing



- Implement Atlantis context on storybook preview and change locale to "es"
- Check InputDate/Mobile/Basic, it should have a different date format
- Check InputTime/Mobile/Basic, it should have a different time format

```diff
diff --git a/.storybook/preview.tsx b/.storybook/preview.tsx
index 7df4c8d5..ae1f6f40 100644
--- a/.storybook/preview.tsx
+++ b/.storybook/preview.tsx
@@ -1,6 +1,7 @@
 import React from "react";
 import { Text } from "@jobber/components/Text";
 import { Content } from "@jobber/components/Content";
+import { AtlantisContext, defaultValues } from "@jobber/components-native";
 import { tokens } from "@jobber/design/foundation";
 import { Header } from "./components/Header";
 import { InlineCode } from "./components/InlineCode";
@@ -71,4 +72,5 @@ export const decorators = [
     </SBProvider>
   ),
   (Story, context) => <StoryDetails Story={Story} context={context} />,
+  (Story) => <AtlantisContext.Provider value={{...defaultValues, locale: "es"}}>{Story()}</AtlantisContext.Provider>,
 ];
```

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

![Random photo of Atlantis](https://loremflickr.com/672/400/atlantis)
